frontend/src/features/public-form/components/FormStartPage/FormBannerLogo.tsx

```diff
@@ -56,15 +80,23 @@ export const FormBannerLogo = ({
   if (!hasLogo) return null
```

Comment:
If there's no logo then we don't actually display this form logout button at all. is that intended? seems a bit strange...

Reply:
good catch. will talk to @pearlyong

Reply:
will always display a logo now on end page (defaults to form logo, follows pearly's new designs)